<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registro</class>
 <widget class="QWidget" name="Registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="leUsuario">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leClave">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leApellido">
     <property name="placeholderText">
      <string>Apellido</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMail">
     <property name="placeholderText">
      <string>Correo electrónico</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbRegistrar">
     <property name="text">
      <string>Registrarse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
